### PR TITLE
Added color field in point themes

### PR DESCRIPTION
**Pull request by goyal1092**
## Description
* Added color field in point themes model, serializer , api and admin
* Added default pallet
* Added schema and data migration

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-299

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [x]  black was run locally (as part of the pre-commit hook)

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/points/migrations/0051_auto_20220916_0731.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            model_name='historicaltheme',
+            name='color',
+            field=colorfield.fields.ColorField(default='#000000', image_field=None, max_length=18, samples=[('#3a70ff', 'theme-1'), ('#993aff', 'theme-2'), ('#ad356d', 'theme-3'), ('#f04f4f', 'theme-4'), ('#ff3a8c', 'theme-5'), ('#ff893a', 'theme-6'), ('#e7bc20', 'theme-7'), ('#48c555', 'theme-8'), ('#2ccaad', 'theme-9'), ('#0a8286', 'theme-10')]),
+        ),
+        migrations.AlterField(
+            model_name='theme',
+            name='color',
+            field=colorfield.fields.ColorField(default='#000000', image_field=None, max_length=18, samples=[('#3a70ff', 'theme-1'), ('#993aff', 'theme-2'), ('#ad356d', 'theme-3'), ('#f04f4f', 'theme-4'), ('#ff3a8c', 'theme-5'), ('#ff893a', 'theme-6'), ('#e7bc20', 'theme-7'), ('#48c555', 'theme-8'), ('#2ccaad', 'theme-9'), ('#0a8286', 'theme-10')]),
+        ),
+    ]
```
Comment (michaelglenister) [inline wazimap_ng/points/migrations/0051_auto_20220916_0731.py:24]: Can the changes here just be included in the AddField for 0050_auto_20220916_0720.py?

File: wazimap_ng/points/migrations/0052_auto_20220916_0733.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.28 on 2022-09-16 07:33
```
Comment (michaelglenister) [inline wazimap_ng/points/migrations/0052_auto_20220916_0733.py:1]: If I add some custom things to a migration I like to change the name of the file but it's not needed just preference

**Review by michaelglenister — APPROVED** [review]
Just minor comments, otherwise it's ready to merge

**Comment by goyal1092** [comment]
Done @michaelglenister

**Pull request merged**